Do not override local modifications by default

wrabcak opened the pull request:
Following PR resolves: 
https://bugzilla.redhat.com/show_bug.cgi?id=1597545

bachradsusi commented:
I have only cosmetic issues with PR:

I would squash d407864 and d9921ca together.

`selinux_all_purge` is not documented in commit message and none of new variables is documented in README.

It could be useful to have also tests to cover new variables and new functionality and to be sure that we don't override system modifications.

Otherwise it looks good.

wrabcak commented:
> I would squash d407864 and d9921ca together.
Fixed.

> selinux_all_purge is not documented in commit message and none of new variables is documented in README.
Fixed.

> It could be useful to have also tests to cover new variables and new functionality and to be sure that we don't override system modifications.

Let's create issue for missing tests.

bachradsusi commented on README.md:
````diff
@@ -45,6 +45,15 @@ roles:
+selinux_fcontexts_purge: 1
+selinux_ports_purge: 1
+selinux_logins_purge: 1
+```
````
There were some comments about using boolean True/False instead of numbers. Should this be the case?

wrabcak replied:
Fixed.

pcahyna replied:
having `selinux_all_purge: true` with the other `*purge` variables in the same yaml example is IMO confusing. Can you split it into two example files, one with `selinux_all_purge`, the other with the others?

pcahyna commented:
[citest bad]

wrabcak commented:
Looks like this PR is ready. Will rebase and merge.
